lib/features/workout/ui/containers/workout_overview.dart

```diff
@@ -0,0 +1,167 @@
+import 'package:flex_workout_mobile/core/common/ui/components/flex_list_tile.dart';
+import 'package:flex_workout_mobile/core/common/ui/components/section.dart';
+import 'package:flex_workout_mobile/core/config/router.dart';
+import 'package:flex_workout_mobile/core/extensions/ui_extensions.dart';
+import 'package:flex_workout_mobile/core/theme/app_layout.dart';
+import 'package:flex_workout_mobile/features/workout/data/models/workout_model.dart';
+import 'package:flutter/material.dart';
+import 'package:material_symbols_icons/symbols.dart';
+
+class WorkoutOverview extends StatelessWidget {
+  const WorkoutOverview({
+    required this.workout,
+    super.key,
+  });
+
+  final WorkoutModel workout;
+
+  @override
+  Widget build(BuildContext context) {
+    return Section(
+      header: 'Overview',
+      subHeader: 'Workout Summary',
+      padding: const EdgeInsets.symmetric(horizontal: AppLayout.p4),
+      body: ListView.separated(
+        shrinkWrap: true,
+        padding: EdgeInsets.zero,
+        physics: const NeverScrollableScrollPhysics(),
+        itemCount: workout.sections.length,
+        itemBuilder: (context, index) {
+          final section = workout.sections[index];
+
+          switch (section) {
+            case final SupersetWorkoutSectionModel supersetSection:
+              return FlexListTile(
+                title: Text(
+                  supersetSection.title,
+                  maxLines: 2,
+                  overflow: TextOverflow.ellipsis,
+                  style: context.typography.bodyMedium.copyWith(
+                    fontWeight: FontWeight.w500,
+                    color: context.colors.foregroundPrimary,
+                  ),
+                ),
+                subtitle: Row(
+                  children: [
+                    Text(
+                      supersetSection.getTotalSets().values.join('-'),
```

Comment:
What happens if the total sets are the same? Would this not return 3-3

Reply:
the added logic to return one total set if the first and last values are the same should resolve this